### PR TITLE
feat(Button): add xs Button size

### PR DESCRIPTION
**Pull request by Marginy605**

### CODE REVIEW

File: src/components/Table/hoc/withTableActions/withTableActions.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ export type TableActionConfig<I> = TableAction<I> | TableActionGroup<I>;
 
 export interface WithTableActionsProps<I> {
     getRowActions: (item: I, index: number) => TableActionConfig<I>[];
-    rowActionsSize?: ButtonSize;
+    rowActionsSize?: Omit<ButtonSize, 'xs'>;
```
Comment (Marginy605) [inline src/components/Table/hoc/withTableActions/withTableActions.tsx:57]: Prop is used for Button and Menu components at the same time. But Menu doesn't support xs size.

File: src/components/Table/hoc/withTableActions/withTableActions.tsx [hunk 2]
```diff
@@ -122,7 +122,7 @@ export function withTableActions<I extends TableDataItem, E extends {} = {}>(
                         disabled={disabled}
                         className={BUTTON_CLASSNAME}
                         onClick={this.handleActionsButtonClick.bind(this, {item, index})}
-                        size={rowActionsSize}
+                        size={rowActionsSize as ButtonSize}
```
Comment (Marginy605) [inline src/components/Table/hoc/withTableActions/withTableActions.tsx:125]: Only for Button component

File: src/components/Table/hoc/withTableActions/withTableActions.tsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ export type TableActionConfig<I> = TableAction<I> | TableActionGroup<I>;
 
 export interface WithTableActionsProps<I> {
     getRowActions: (item: I, index: number) => TableActionConfig<I>[];
-    rowActionsSize?: ButtonSize;
+    rowActionsSize?: Exclude<ButtonSize, 'xs'>;
```
Comment (Marginy605) [inline src/components/Table/hoc/withTableActions/withTableActions.tsx:57]: Prop is used for Button and Menu components at the same time. But Menu doesn't support xs size.

File: src/components/Button/Button.scss [hunk 4]
```diff
@@ -85,6 +85,46 @@ $block: '.#{variables.$ns}button';
...
+                }
+
+                &_side_left ~ #{$block}__text {
+                    margin-left: 22px;
+                }
+
+                &_side_right {
+                    margin-right: 6px;
+                    width: var(--yc-button-icon-size);
+                }
```
Comment (amje) [inline src/components/Button/Button.scss:113]: No need to set these styles, `width: var(--yc-button-height);` does all the work. Same relates to &_side_left

File: src/components/Table/hoc/withTableActions/withTableActions.tsx [hunk 5]
```diff
@@ -52,9 +52,14 @@ export interface TableActionGroup<I> {
 
 export type TableActionConfig<I> = TableAction<I> | TableActionGroup<I>;
 
+/**
+ * common sizes for Menu and Button
+ */
+export type RowActionsSize = 's' | 'm' | 'l' | 'xl';
```
Comment (amje) [inline src/components/Table/hoc/withTableActions/withTableActions.tsx:58]: `TableRowActionsSize` would be better and can be exported safely

**Review by amje — CHANGES_REQUESTED** [review]

**Review by korvin89 — APPROVED** [review]

**Review by amje — APPROVED** [review]

**Pull request merged**